lab2/README.md

```diff
@@ -0,0 +1,59 @@
+# Lab 2
+
+Your goal for this lab is to make an app that uses two APIs, where one of them has to be an Android API. 
+
+You will have 2 weeks to complete this lab. **Lab 2 is due at 6:30pm on Thursday, October 27th.** This lab will be done in groups of either two or three. There should be two groups of three. You should work with new people on this lab.
+
+[Here](./RUBRIC.md) is the rubric.
+
+## Deliverables
+We require these two things to be turned in:
+* Your proposal + Wireframes + Design Explanation (Sun Oct 16)
+* Your app (Thu Oct 27)
+
+## MVP
+The only requirement for this lab is that you use two APIs. One API has to be an Android API that interacts with your phone's hardware (see below for examples) while the other one can be either an Android or web API. All other requirements are scoped by your team in your proposal.
+
+These are acceptable Android APIs that you can use. Let us know ahead of time if you want to work with a different API.
+
+* [Camera](https://developer.android.com/reference/android/hardware/camera2/package-summary.html)
+* [Sensors](https://developer.android.com/guide/topics/sensors/sensors_overview.html)
+  * Motion
+  * Environmental
+  * Position
+* [Infrared Emitter](https://developer.android.com/reference/android/hardware/ConsumerIrManager.html)
+* [Near Field Communication (NFC)](https://developer.android.com/guide/topics/connectivity/nfc/index.html)
+* [Bluetooth](https://developer.android.com/guide/topics/connectivity/bluetooth.html)
+* [Touch gestures](https://developer.android.com/training/gestures/index.html)
+
+## Proposal
+Before the end of today’s class (Thur Oct 13) each team will turn in a proposal. Include the following information:
+
+* The names of everyone in your group.
+* The two APIs.
+* The feature(s) you plan on implementing as:
+  * MVP
+  * Stretch goals
+* The name of the git repo `username/RepoNamo` you will be using (you should make a new one).
```

Comment:
You can put this in the survey below instead of in the proposal, because we need to know where to find the proposal anyway